cmd/util/cmd/read-execution-state/list-accounts/cmd.go

```diff
@@ -99,11 +99,15 @@ func run(*cobra.Command, []string) {
 	st := state.NewState(ldg)
 	accounts := state.NewAccounts(st)
 	finalGenerator, err := state.NewStateBoundAddressGenerator(st, chain)
-
 	if err != nil {
 		log.Fatal().Err(err).Msgf("cannot get current address state")
 	}
 
+	err = st.Commit()
```

Comment:
Is this a bugfix?

Reply:
Not exactly, just moved the state commit to the function that created the state.